src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -21114,32 +21090,26 @@ static void F<T>()
     {
         T s = default;
         s.ToString();
-        T? t = default;
-        t.ToString();
     }
 }";
 
             var comp = CreateCompilation(
                 new[] { source, NonNullTypesTrue, NonNullTypesAttributesDefinition },
                 parseOptions: TestOptions.Regular8);
             comp.VerifyDiagnostics(
+                // (5,15): warning CS8625: Cannot convert null literal to non-nullable reference or unconstrained type parameter.
```

Comment:
>null literal [](start = 58, length = 12)

Can we improve the diagnostic, or maybe add a PROTOTYPE marker? The diagnostic mentions "null literal" but there is no `null` in code.

Reply:
I'll add a prototype comment, seems out of scope for this PR.

---
In reply to: [207680174](https://github.com/dotnet/roslyn/pull/28956#discussion_r207680174) [](ancestors = 207680174)